CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## Main
+* [5626](https://github.com/grafana/loki/pull/5696) **paullryan** Ignore and move one when cloudflare reports errors about too early logs.
```

Comment:
`move one` should be `move on`, but maybe an entry like `don't block scraping of new logs from cloudflare within promtail if an error is received from cloudflare about too early logs`? `too early` also sounds a bit off to me but from the looks of it that's the actual message they report from their end right?

Reply:
Yes that is the language from cloudflare. Updated per your suggestion.